### PR TITLE
Update viewed state invoices to overdue

### PR DESCRIPTION
**Pull request by shalapatil**
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [ ] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by supriya3105** [comment]
@apoorv1316 Can you test this by running it on local.

File: app/services/update_invoice_status_to_overdue_service.rb [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 class UpdateInvoiceStatusToOverdueService
   def process
-    Invoice.sent.where("due_date < ?", Date.current).update_all(status: "overdue")
+    Invoice
+      .where(status: ["viewed", "sent"])
+      .where("due_date < ?", Date.current)
+      .update_all(status: "overdue")
```
Comment (akhilgkrishnan) [inline app/services/update_invoice_status_to_overdue_service.rb:8]:
```suggestion
    Invoice
      .where(status: ["viewed", "sent"], due_date: ...Date.current)
      .update_all(status: "overdue")
```

**Review by akhilgkrishnan — CHANGES_REQUESTED** [review]
@shalapatil Requested one change

**Review by apoorv1316 — APPROVED** [review]

**Comment by apoorv1316** [comment]
> @apoorv1316 Can you test this by running it on local.

@supriya3105 tested on local.

**Review by akhilgkrishnan — APPROVED** [review]
LGTM

**Pull request merged**